src/main/java/frc/robot/Robot.java

```diff
@@ -148,6 +148,9 @@ public class PDHValues {
     Current PORT0 = Amps.of(PDH.getCurrent(0));
     Current PORT1 = Amps.of(PDH.getCurrent(1));
     Current PORT2 = Amps.of(PDH.getCurrent(2));
```

Comment:
These can be deleted since you already have a named version. Same for all others